index.html

```diff
@@ -2992,174 +2882,221 @@ <h3>Other Discovery Methods</h3>
-						and is updated by each applicable transformation.</p>
-					<ol style="list-style-type: lower-alpha;">
-						<li id="processing-expansion-arrays">
-							<p>(<a href="#value-array"></a>) If <var>term</var> expects an <a href="#value-array"
-									>array</a> but <var>current</var> contains a string or object <var>value</var>,
-								convert <var>current</var> to an array and push <var>value</var> onto it.</p>
-							<details>
-								<summary>Explanation</summary>
-								<p>A number of terms require their values to be arrays, but, for the sake of
-									convenience, authors are allowed to use a single value instead of a one element
-									array. For example,</p>
-								<pre class="example">{
+							<li id="processing-normalize-arrays">
+								<p>(<a href="#value-array"></a>) If <var>term</var> expects an <a href="#value-array"
+										>array</a> and <var>termValue</var> is a <a>single value</a> or <a
+										href="https://infra.spec.whatwg.org/#ordered-map">map</a>, set
+										<var>normalized</var> to «&#160;<var>termValue</var>&#160;».</p>
+								<details>
+									<summary>Explanation</summary>
+									<p>A number of terms require their values to be arrays, but, for the sake of
+										convenience, authors are allowed to use a single value instead of a one element
+										array. For example,</p>
+									<pre class="example">{
     …
     "name"   : "Et dukkehjem",
     "author" : "Henrik Ibsen",
     …
 }</pre>
-								<p>yields:</p>
-								<pre class="example">{
+									<p>yields:</p>
+									<pre class="example">{
     …
     "name"   : ["Et dukkehjem"],
     "author" : ["Henrik Ibsen"],
     …
 }</pre>
-							</details>
-						</li>
-
-						<li id="processing-expansion-entities">
-							<p>(<a href="#value-entity"></a>) If <var>term</var> expects an <a href="#value-entity"
-									>entity</a> but the value of <var>current</var> is a string <var>str</var>, convert
-									<var>current</var> to an object with the following properties:</p>
-							<ul>
-								<li><p><var>type</var> &#8211; set to an array with the value
-									"<code>Person</code>";</p></li>
-								<li><p><var>name</var> &#8211; set to <var>str</var>.</p></li>
-							</ul>
-							<p>If typeof(<var>current</var>) is <code>Array</code>&#160;[[!ecmascript]], perform this
-								step on each element of the array.</p>
-							<details>
-								<summary>Explanation</summary>
-								<p>Authors, editors, etc., are expected to be explicitly designed as an object of type
-										<code>Person</code>, but, for the sake of convenience, only their name has to be
-									specified. For example:</p>
-								<pre class="example">{
+								</details>
+							</li>
+
+							<li id="processing-normalize-entities">
+								<p>(<a href="#value-entity"></a>) If <var>term</var> expects an <a href="#value-entity"
+										>entity</a>:</p>
+								<ol>
```

Comment:
Strictly speaking, this is incorrect. Per our definition, e.g., `author` _never_ expects and entity; it expects an _array of entities_. I presume that what we have to put there in the 'If' clause.

Reply:
Yes, I don't think we ever require a single entity, but I suppose the prose fix to leave in place just in case we ever do is:

> If term expects a single entity or array of entities